src/components/organisms/UiList/_internal/UiListItem.vue

```diff
@@ -42,25 +42,23 @@
   </li>
 </template>
 
-<script lang="ts">
-export default { inheritAttrs: false };
-</script>
-
 <script setup lang="ts">
 import {
-  computed,
   ref,
+  computed,
+  useAttrs,
+  defineAsyncComponent,
+  defineExpose,
```

Comment:
[@vue/compiler-sfc] `defineExpose` is a compiler macro and no longer needs to be imported ☺️